### PR TITLE
Add an 'query-external-data' example 

### PR DESCRIPTION
**Pull request by jkammerer**
that demonstrates several possibilities that became available with the new `external()` functionality.

### CODE REVIEW

File: Community-Supported/query-external-data/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# Get started
+
+## __Prerequisites__
+
+To run the script, you will need:
+
+- a computer running Windows, macOS, or Linux
+
+- Python 3.6 or 3.7
```
Comment (jonas-eckhardt) [inline Community-Supported/query-external-data/README.md:17]: Let's say sth like "Python (3.7 or newer)"

File: Community-Supported/query-external-data/query_external_data.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            connection.execute_command(command_1)
+            
+            print("table content:")
+            print_list(connection.execute_list_query("SELECT * FROM low_prio_orders"))
+            print()
+                
+         
+            print("\nScenario 2: Query multiple external data sources in one query.")
+            # This query reads data from a parquet and a CSV file and joins it. Note that, for CSV files, the schema of the file
+            # has to be provided and currently cannot be inferred form the file directly.
```
Comment (jonas-eckhardt) [inline Community-Supported/query-external-data/query_external_data.py:58]: Can you please add "See the "DESCRIPTOR" argument"

**Review by jonas-eckhardt — COMMENTED** [review]
Can you also add a sample for CREATE EXTERNAL TABLE?

File: Community-Supported/query-external-data/query_external_data.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+
+
+if __name__ == '__main__':
+    try:
+        run_hyper_query_external()
+
+    except HyperException as ex:
+        print(ex)
+        exit(1)
```
Comment (vogelsgesang) [inline Community-Supported/query-external-data/query_external_data.py:95]: unnecessary. Python by default already prints uncaught exceptions

File: Community-Supported/query-external-data/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+1. Open a terminal and activate the Python virtual environment (`venv`).
+
+1. Navigate to the folder where you installed the sample.
+
+1. Run the Python script:
+   
+   **python query_external_data.py**
+
+   It will read the `orders_10rows.parquet` file from the working directory and create a new Hyper database
+   named `orders.hyper` with a table named "orders", which will contain the 10 rows copied from the Parquet file.
```
Comment (vogelsgesang) [inline Community-Supported/query-external-data/README.md:35]: looking at the script, the script does much more. We should restructure the README to actually point out all the contributions...

**Pull request merged**